lib/ssz_ex.ex

```diff
@@ -18,6 +18,7 @@ defmodule LambdaEthereumConsensus.SszEx do
   @zero_chunk <<0::size(@bits_per_chunk)>>
   @zero_hashes ZeroHashes.compute_zero_hashes()
 
+  @compile {:inline, hash: 1}
   @spec hash(iodata()) :: binary()
```

Comment:
shouldnt this function live outside of SSZ? I see it's being used in other places too, like shuffling

Reply:
Maybe. We just added it here since adding a new module for this function seemed like too much